Content.Shared/_Moffstation/Paper/Components/ForgeSignatureComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._Moffstation.Paper.Components;
+
+[RegisterComponent, NetworkedComponent]
+[AutoGenerateComponentState]
+public sealed partial class ForgeSignatureComponent : Component
+{
+    /// <summary>
+    /// The the signature written when a paper is signed
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField]
+    [AutoNetworkedField]
+    public string Signature = "";
```

Comment:
```suggestion
    [DataField, AutoNetworkedField]
    public string Signature = "";
```
VV with r/w is implied on `DataField`